kubernetes/config/kube_config_yaml.c

```diff
@@ -430,25 +430,34 @@ int kubeyaml_load_kubeconfig(kubeconfig_t * kubeconfig)
 {
     static char fname[] = "kubeyaml_load_kubeconfig()";
 
-    /* Set a file input. */
+    /* Set a file input or use the provided buffer. */
     FILE *input = NULL;
```

Comment:
Add a test here to verify that only one of `fileName` and `buffer` is non-null.

Reply:
Done.